src/IscsiLioData.ycp

```diff
@@ -225,6 +225,18 @@ boolean IsTpgActive( string tgt, integer tpg )
     return( data["tgt",tgt,tpg,"ep","enabled"]:false );
     }
 
+boolean LogExecAuthCmd( string cmd, string user, string password )
```

Comment:
I have used this name because it fits to existing 'LogExecCmd' function and to my understanding purpose is indeed executing and logging of command.